infra/isra.tf

```diff
@@ -7,7 +7,7 @@ module "iam_eks_role" {
   oidc_providers = {
     eks = {
       provider_arn               = "${module.eks.oidc_provider_arn}"
-      namespace_service_accounts = ["${var.k8s_namespace}:${var.service_account_name}"]
+      namespace_service_accounts = ["k8s-alb-ns:${var.service_account_name}"]
```

Comment:
Same for svc account, it won't change, no need for variable, update helm as well.